201-web-app-asev2-ilb-create/azuredeploy.parameters.json

```diff
@@ -15,7 +15,7 @@
             "value": "changeme"
         },
         "internalLoadBalancingMode": {
-            "value": 3
+            "value": "changeme"

```

Comment:
remove this and use a valid default value in the template

Reply:
Updated to a valid default value per schema